### PR TITLE
Update to webpack 5

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

*If this is a regression, consider adding it to #1898!*

# Description

Please include a summary of the change and which issue was fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Type of change

Include at least one commit in your PR that marks the change-type. This can be either specified through a Change-type footer, or by adding the change-type as a prefix to the commit, i.e. minor: Add some new feature. This is so the PR can be automatically versioned and a changelog generated for it by using versionist. Check out [semver](https://semver.org/) for a detailed explanation of the different possible change-type values.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### CODE REVIEW

File: tsconfig.json
Comment (Page-) [inline tsconfig.json:4]: This was previously blocked by webpack 4 not supporting es2021 syntaxThis was previously blocked by webpack 4 not supporting es2021 syntax

**Review by pipex — APPROVED** [review]
LGTM

**Comment by pipex** [comment]
I already tested on a device before merging. Forgot to mention it here

**Pull request merged**
